src/services/db/GitHubService.js

```diff
@@ -284,6 +284,27 @@ class GitHubService {
     return { treeSha, currentCommitSha }
   }
 
+  async getLatestCommitOfBranch(sessionData, branch) {
```

Comment:
Discussed offline, would prefer to centralise everything so that when refactoring to TypeScript, it is easier to find all the functions that we need to refactor in one place.